pnpm-lock.yaml

```diff
@@ -42,7 +42,7 @@ importers:
         version: link:packages/tools/changelog-generator-wrapper
       '@fluid-tools/build-cli':
         specifier: ^0.55.0
-        version: 0.55.0(@types/node@18.19.67)(encoding@0.1.13)(typescript@5.4.5)(webpack-cli@5.1.4)
+        version: 0.55.0(@types/node@22.10.1)(encoding@0.1.13)(typescript@5.4.5)(webpack-cli@5.1.4)
```

Comment:
I'm still curious why for some people/environments the node types version gets updated like this. Are you using Node22 locally?

Reply:
Nope, I'm on Node20. I'm not sure either.